libs/styles/rounded-corners.css

```diff
@@ -1,29 +1,36 @@
 :root {
   /* border-radius sizes */
+  --rounded-corners: 0;
   --s-rounded-corners: 4px;
   --m-rounded-corners: 8px;
   --l-rounded-corners: 16px;
   --full-rounded-corners: 50%;
 }
 
+.rounded-corners,
+.rounded-corners > .background img,
+.rounded-corners-image .foreground .image picture img {
+  border-radius: var(--rounded-corners);
+}
+
```

Comment:
Thanks for the updates on this. 
I was thinking the default of `rounded-corners` to have a value and just added to either the `s-` or `m-` instances. It doesn't make sense to have it at all if the value is just 0. My thinking is that the most often use case shouldn't have to denote a size, but just rounded-corner: default 8. etc.